chef/data_bags/crowbar/migrate/neutron/022_seperate_ml2_drivers.rb

```diff
@@ -0,0 +1,57 @@
+def upgrade ta, td, a, d
+  if a['networking_mode']
+    a['ml2_type_drivers'] = a['networking_mode']
+  else
+    a['ml2_type_drivers'] = ta['ml2_type_drivers']
+  end
+
+  if a['networking_plugin'] == 'linuxbridge'
+    a['networking_plugin'] = 'ml2'
+    a['ml2_mechanism_drivers'] = 'linuxbridge'
+  elsif a['networking_plugin'] == 'openvswitch'
+    a['networking_plugin'] = 'ml2'
+    a['ml2_mechanism_drivers'] = 'openvswitch'
```

Comment:
> Same here.

Done in https://github.com/toabctl/barclamp-neutron/commit/e031434a8db081f91d103898fb8c5486eaf69621